### PR TITLE
Convert Lisk Hub to Progressive Web Application - Close #509

### LINKED ISSUE #509: Convert Lisk Hub to Progressive Web Application (PWA)
### Expected behaviour
Update/Tune/Modify **Lisk Hub** configuration/build processes and code to make a **PWA** from **Lisk Hub**. It will allow installing **Lisk Hub** on mobile devices directly from the site and use as an app.

**why:**
Progressive Web Apps take advantage of new technologies to bring the best of mobile sites & native apps to users. 
**pros**:
 - No installation required
 - No application update required
 - Offline use
 - Network connection is secure
 - Quick loading
 - Home screen icon
 - Single codebase for all platforms
 - Code can run in a background process
 - Custom splash screen / address bar matched a brand color

**cons**:
 - More limited hardware access
 - Not all browsers are supported (Works fine on Android devices, But `IOS` will allow using PWA soon)
 - Still a new technology 

### Actual behaviour
**Lisk Hub** is not a PWA

### Steps to reproduce
None

### PR DESCRIPTION
**Pull request by alepop**
### What was the problem?
Lisk Hub is not a PWA
### How did I fix it?
Add and configured next packages:
[webpack-pwa-manifest](https://github.com/arthurbergmz/webpack-pwa-manifest) - to generate a manifest file
[offline-plugin](https://github.com/NekR/offline-plugin) - to configure service worker and app cache 

### How to test it?
1) Build project with `production` flag.
2) Run server with Lisk Hub
3) Chrome on a computer:  `Dev tools` -> `Application` -> `Manifest` -> `Add to homescreen`
4) Turn off internet connection
5) Refresh page

To check cache update:
1) Change source file (add `alert(1)` for example)
2) Build new version
3) Reload the webpage
4) Close the tab of the webpage (make sure there are no more open tabs of this website)
5) Open that webpage again in a new tab
6) Check `alert(1)`

I haven't any `Android` device to check how it will work on it, but this is what I got on `iPhone` with the latest ios `11.3`:

### Todo?
- [ ] Add logo with high resolution and convert to small sizes.
- [ ] Add message when a device is `offline`?
- [ ] Add message when the new version is available?
- [ ] Add splash into the manifest file for mobile devices?

Any suggestion?
### Review checklist
- The PR solves #509
- All new code follows best practices

### CODE REVIEW

**Comment by alepop** [comment]
Lighthouse audit score:
screen shot 2018-04-06 at 11 53 09

File: config/webpack.config.react.js [hunk 1]
```diff
@@ -131,6 +133,44 @@ module.exports = {
       ],
     }),
     new ContextReplacementPlugin(/moment[/\\]locale$/, new RegExp(langRegex)),
+    new WebpackPwaManifest({
+      name: 'Lisk Hub',
+      short_name: 'Lisk Hub',
+      description: '🖥 Graphical User Interface (GUI) to manage Lisk IDs, tokens and your sidechains ',
+      theme_color: '#005a90',
+      background_color: '#013165',
```
Comment (slaweet) [inline config/webpack.config.react.js]: The background_color on splash screen IMO doesn't look too good:
screenshot_2018-04-09-15-08-27-698
What about using something like what we use when non-pwa is loading?
screenshot_2018-04-09-15-11-46-605

File: config/webpack.config.react.js [hunk 2]
```diff
@@ -131,6 +133,44 @@ module.exports = {
...
+      orientation: 'portrait',
+      display: 'standalone',
+      start_url: '/',
```
[the remark this answers is not in the document]
Comment (alepop) [reply config/webpack.config.react.js]: got it

File: config/webpack.config.react.js (hunk 1 above)
Comment (alepop) [reply config/webpack.config.react.js]: @slaweet already provide some requests to LiskHQ designers to get splash screen image and correct colors palet.

**Comment by alepop** [comment]
@slaweet closed for security reasons?

**Comment by slaweet** [comment]
I didn't close it consciously, I just deleted the base branch (0.5.0) because the 0.5.0 release is out. :-D
But yes, in the current situation, there is no point in continuing PWA. We can re-open it if Hub is hosted again.

**Pull request closed without merging**